content/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -411,8 +415,8 @@ or _flapping_. It's similar to the concept of _hysteresis_ in cybernetics.
 ## 工作量规模的稳定性 {#flapping}
 
 在使用 HorizontalPodAutoscaler 管理一组副本的规模时，由于评估的指标的动态特性，
-副本的数量可能会经常波动。这有时被称为 **抖动（thrashing）** 或 **波动（flapping）**。
-它类似于控制论中的 **滞后（hysteresis）** 概念。
+副本的数量可能会经常波动。这有时被称为**抖动（thrashing）**或**波动（flapping）**。
+它类似于控制论中的**滞后（hysteresis）**概念。
```

Comment:
This doesn't work as expected. Please check the preview.
Removing this extra spaces may confuse the buggy interpreter.

Reply:
fixed